src/js/events/MesosStateActions.js

```diff
@@ -21,10 +21,11 @@ var MesosStateActions = {
             });
             resolve();
           },
-          error(e) {
+          error(xhr) {
             AppDispatcher.handleServerAction({
               type: ActionTypes.REQUEST_MESOS_STATE_ERROR,
-              data: e.message
+              data: xhr.message,
```

Comment:
Shouldn't we parse this through `RequestUtil.getErrorFromXHR(xhr)` like we do other places?